common/src/main/java/com/wynntils/features/inventory/CustomBankQuickJumpsFeature.java

```diff
@@ -310,9 +314,9 @@ private List<Integer> parseStringToDestinations(String destinationsStr, Searchab
                 List<Integer> defaultValues;
 
                 switch (containerType) {
-                    case BANK -> defaultValues = QUICK_JUMP_DESTINATIONS;
+                    case ACCOUNT_BANK -> defaultValues = QUICK_JUMP_DESTINATIONS;
                     case BLOCK_BANK -> defaultValues = BLOCK_BANK_DESTINATIONS;
-                    case BOOKSHELF, MISC_BUCKET -> defaultValues = HOUSING_DEFAULT_DESTINATIONS;
+                    case BOOKSHELF, MISC_BUCKET, CHARACTER_BANK -> defaultValues = HOUSING_DEFAULT_DESTINATIONS;
```

Comment:
`HOUSING_DEFAULT_DESTINATIONS` could be renamed to a generic "10-page destination" variable. Not sure how to call it though..

Reply:
How about `MINIMUM_DEFAULT_DESTINATIONS`?

Reply:
Not really a minimum value though. Perhaps just DEFAULT_DESTINATIONS could work.